bspump/avro/sink.py

```diff
@@ -47,12 +49,11 @@ def __init__(self, app, pipeline, id=None, config=None):
 
 		self.SchemaFile = self.Config.get('schema_file')
 		if self.SchemaFile is None:
-			L.error('schema is not defined')
-			raise FileNotFoundError
+			raise FileNotFoundError('Avro schema is not defined')
 
 		if self.SchemaFile is not None:
```

Comment:
Use unified function `load_avro_schema()` - this applies for any other location.